xml/System.Windows.Input.Manipulations/Manipulator2D.xml

```diff
@@ -154,7 +154,12 @@
         <returns>
           <see langword="true" /> if the two <see cref="T:System.Windows.Input.Manipulations.Manipulator2D" /> objects have the same value; otherwise, <see langword="false" />.</returns>
         <remarks>
-          <format type="text/markdown"><![CDATA[The equivalent method for this operator is <xref:System.Windows.Input.Manipulations.Manipulator2D.Equals%2A?displayProperty=nameWithType>]]>.</format>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+ The equivalent method for this operator is <xref:System.Windows.Input.Manipulations.Manipulator2D.Equals%2A?displayProperty=nameWithType>
+
+ ]]>.</format>
```

Comment:
The period here should be at the end of the `<xref`, inside the `]]>` not outside.